DMX.Gatekeeper.Api/Controllers/HomeController.cs

```diff
@@ -0,0 +1,17 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using Microsoft.AspNetCore.Mvc;
+
+namespace DMX.Gatekeeper.API.Controllers
```

Comment:
Make sure the namespace has `namespace DMX.Gatekeeper.Api.Controllers`

Reply:
Ahh good eye-- that's one I missed